nearcore/src/runtime/mod.rs

```diff
@@ -444,7 +441,7 @@ impl NightshadeRuntime {
         is_first_block_with_chunk_of_version: bool,
         states_to_patch: Option<Vec<StateRecord>>,
     ) -> Result<ApplyTransactionResult, Error> {
-        let _span = tracing::debug_span!(target: "runtime", "process_state_update").entered();
+        let _span = tracing::debug_span!(target: "runtime", "Process state update").entered();
```

Comment:
Do we have our casing conventions for spans documented somewhere? If not, could you come up with one and add it to https://github.com/near/nearcore/blob/master/docs/style.md#tracing ?

Reply:
To bikeshed this a bit, I feel the most reasonable convention is to just use the function name? I don't think span names need to be "human readable", but it'd be cool if they are obviously greppable.

Reply:
> conventions

Documented, but I suggest to allow not only `snake_case` names. For example, span named "NetworkClientMessages" makes more sense than "handle" or "NetworkClientMessages::handle".

Reply:
> full event formatter

Disabled synthetic events. I can already see that the full formatter doesn't output span duration to the logs. We may want to keep those durations in the logs until a distributed tracing system is ready.

Reply:
> Documented, but I suggest to allow not only snake_case names. For example, span named "NetworkClientMessages" makes more sense than "handle" or "NetworkClientMessages::handle".

I feel like `NetworkClientMessages` thing might want to be a KV, rather than a span name. Perhaps a most similar analogy here is with the HTTP server, which will typically produce spans that are of this shape:

```
handle_request method=GET endpoint=/sandwich header.sudo=no header.user-agent=Jeff ...
```

instead of

```
GET endpoint=/...
```

or 

```
get_sandwich ...
```

Generic span names has a couple benefits: it minimizes the duplication in instrumentation (you only really need to instrument the function that dispatches the messages), but also it gives you more flexibility to extend the instrumentation in ways that avoid making historical data disjoint.

That said, I feel like this doesn't need to be solved now. I believe that experience will best inform what works best for us. It might very well be that having spans named `NetworkClientMessages` is the best approach for us, all rules and thumbs be damned.

> Disabled synthetic events. I can already see that the full formatter doesn't output span duration to the logs. We may want to keep those durations in the logs until a distributed tracing system is ready.

Right, I wasn't suggesting that we should disable the synthesized events right now. My intention with the comment was to make a forward-looking statement for what we will want to do eventually.